### DISCUSSION TITLE
CefSharp Release timing for 126.1.5+g59e700b+chromium-126.0.6478.26 / Chromium 126.0.6478.26?

### DISCUSSION THREAD

**Discussion by TCabanCNX** [cefsharp/CefSharp#4823] [Q&A]
@amaitland 

Hello Alex - I'm hoping you might be able to share some info on when a Development Build of CefSharp will be available that is built on 126.1.5+g59e700b+chromium-126.0.6478.26 / Chromium 126.0.6478.26?

I've been in touch with the Chromium and CEF teams to get to this point - and I have a 3rd party Developer (Genesys) that relies on CEFSharp for their WebRTC screen recording functionality.  The 126 build mentioned above has a specific fix for a screen flickering problem that we are looking to have Genesys fix.

Genesys has shared that they are dependent upon CEFSharp incorporating this version in order to provide us with a fix.

Hoping you can provide me with some insight into the timeline, so I can manage my end-customer's expectations.

Thanks.

Tom C.

**Comment by amaitland** [comment]
Hi Tom,

> Development Build of CefSharp will be available that is built on 126.1.5+g59e700b+chromium-126.0.6478.26 / Chromium 126.0.6478.26?

Generally I don't upgrade to **beta** `Chromium` builds unless there's major breaking API changes that require `CefSharp` code changes. Typically I wouldn't be looking to upgrade to M126 until it hit `early stable` at a minimum. Then it depends when CEF upgrades and builds becoming available on the Spotify build server. So I wouldn't expected anything for a few weeks

I'd suggest downloading the `CEF Sample Application` from https://cef-builds.spotifycdn.com/index.html and validating the fix works as expected.

Alex

**Reply by amaitland** [reply]
CEF has now upgraded to a stable `126` version, hopefully builds will become available in the next few days. I will try and release a new version next weekend (I cannot make any promises though).

**Comment by TCabanCNX** [comment]
Hi Alex

FYI - I've just been in touch with the CEF Open Source team, and they have shared that 126. is expected to progress to the Stable Build next Tuesday (June 11th).

So my expectations are properly managed - how long does it typically take the CEFSharp build process to reach a point where a beta or Stable build is released?

Tom Caban
Sr Director - IT Service Delivery
West Warwick RI, USA
+1 401 952-3502
[Concentrix logo]
concentrix.com<https://www.concentrix.com/?utm_source=outlook-email&utm_medium=email>

**Comment by TCabanCNX** [comment]
Hi Alex

Wanted to check in to see if you are still tracking towards a release this weekend of CEFSharp based on 126?

I'm going into a client meeting this morning - and having a current update would be super helpful.

Thanks!

Tom Caban
Sr Director - IT Service Delivery
West Warwick RI, USA
+1 401 952-3502
[Concentrix logo]
concentrix.com<https://www.concentrix.com/?utm_source=outlook-email&utm_medium=email>
